### ISSUE TITLE
Customizable max-pods-per-node

### ISSUE THREAD

**Issue by davidgiga1993** [terraform-aws-modules/terraform-aws-eks#1621]
## Is your request related to a new offering from AWS?
No

## Is your request related to a problem? Please describe.
Yes, when using another CNI for example canico the default limit doesn't make sense anymore and should be configurable

## Describe the solution you'd like.
Make the max pods per node a parameter

## Describe alternatives you've considered.
Not using terraform, but it would be a pretty big step back

## Additional context
Same request in the aws provider repo: https://github.com/hashicorp/terraform-provider-aws/issues/20115

**Comment by stevehipwell** [comment]
As #1577 has been merged this can probably be closed? It's worth noting that this was always possible via custom userdata, but it's much easier now.

```hcl
mng = {
  ...

  bootstrap_env = {
    USE_MAX_PODS="false"
  }

  kubelet_extra_args = "--max-pods 110"

  ...
}
```

**Comment by daroga0002** [comment]
I opened https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1667 to update one example

**Comment by antonbabenko** [comment]
This issue has been resolved in version 17.24.0 :tada:
